src/pages/Dashboard/index.tsx

```diff
@@ -83,10 +85,12 @@ export default function Dashboard() {
 							<p>Nenhum dado encontrado para estação selecionada.</p>
 						)
 					) : (
-						<p>Carregando...</p>
+						<LoadingContainer>
+							<Loading />
+						</LoadingContainer>
 					)}
 				</CardContainer>
-				{!isSimpleUser && (
+				{!isLoading && isSimpleUser && (
```

Comment:
Acho que aqui seria `!isSimpleUser`, pq só vai poder ver o botão de cadastrar estação se estiver logado e for algum admin, mas a gente corrige numa próxima PR!